docs/devices/E2123.md

```diff
@@ -69,11 +69,13 @@ Pair the sensor to Zigbee2MQTT by quickly pressing the pair button 4x to get it
   * `dot_<1|2>_long_release`
 <!-- Notes END: Do not edit below this line -->
```

Comment:
Please do not edit below this line

Reply:
I totally overlooked this note, thanks for pointing it out!

Also fixed the heading levels for the notes.